lib/policies/terminate/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  policy: require('./terminate'),
+  schema: {
+    type: 'object',
+    properties: {
+      statusCode: { type: 'number' },
+      message: { type: 'string' }
```

Comment:
I know you responded me already on default values for schema
https://github.com/ExpressGateway/express-gateway/pull/530#discussion_r151138385

but I didn't understand what you meant.